dev/io.openliberty.io.smallrye.config/src/io/openliberty/microprofile/config/internal/extension/OLSmallRyeConfigProviderResolver.java

```diff
@@ -136,6 +133,8 @@ public void applicationStopping(ApplicationInfo appInfo) {
             for (Config config : configsToRelease) {
                 releaseConfig(config);
             }
+
+            dumpCache();
```

Comment:
I'm wondering whether to always have this when trace is enabled. Thoughts?

Reply:
oh, ignore my comments since this can be quite chatty.

Reply:
I have tried to tune this a little ... added some more cache debug trace but also made ConfigWrapper trivial which cuts the trace down quite a bit